util-cgroups/src/measure/v2.rs

```diff
@@ -142,3 +142,250 @@ mod common {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Should these be in integration tests? It's not testing the internals of the lib, just the public interface.